src/GovernanceStaker.sol

```diff
@@ -761,10 +760,7 @@ abstract contract GovernanceStaker is INotifiableRewardReceiver, Multicall {
     uint256 _depositNewEarningPower,
     uint256 _totalEarningPower
   ) internal pure returns (uint256 _newTotalEarningPower) {
-    if (_depositNewEarningPower >= _depositOldEarningPower) {
```

Comment:
was this changed because of a different finding?

Reply:
Good question

Reply:
Yes, it was do you think we should keep it the same?